frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -426,11 +432,13 @@ export function DashCardVisualization({
     return (
       <Flex align="center" justify="flex-end">
         {inlineParameters.length > 0 && (
-          <DashboardParameterList
-            parameters={inlineParameters}
-            isSortable={false}
-            isFullscreen={isFullscreen}
-          />
+          <Box className={S.InlineParametersContainer}>
+            <DashboardParameterList
+              parameters={inlineParameters}
+              isSortable={false}
+              isFullscreen={isFullscreen}
+            />
+          </Box>
```

Comment:
Would it make sense to pass in a className to `DashboardParameterList` instead of adding an extra div just for that? Also it'd make the CSS a tiny bit simpler (by removing the need for ` > *`).